lib/safe-pg-migrations/base.rb

```diff
@@ -53,14 +53,24 @@ def close_alternate_connection
       @alternate_connection = nil
     end
 
-    ruby2_keywords def say(*args)
+    def say(message, subitem = false, sensitive: false)
       return unless current_migration
 
-      current_migration.say(*args)
+      sensitive_logger = SafePgMigrations.config.sensitive_logger
+
+      return current_migration.say message, subitem unless sensitive_logger && sensitive
```

Comment:
I feel like it would be simpler to have a default logger for `sensitive_logger` (that would be the regular logger), so the logic would be:
```rb
if sensitive
  SafePgMigrations.config.sensitive_logger.info(...)
else
  current_migration.say message, subitem
end
```

Reply:
What happens if we don't configure a sensitive logger and the information that we log is sensitive btw?

Reply:
Regarding the second question: same as before, data is logged in STDOUT. Though we have a flag to only display PID of this kind of data.

Reply:
It was a bit mixed up because current_migration is not available when the config is defined. For this reason, I've extracted all logs inside the new helper `Logger` and defined methods there